@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -301,7 +307,14 @@ export class TileGeometryLoader {
             group: TileTaskGroups.CREATE,
             getPriority: this.getPriority.bind(this),
             isExpired: () => {
-                return this.m_state !== TileGeometryLoaderState.CreationQueued;
+                if (this.m_state !== TileGeometryLoaderState.CreationQueued) {
```

Comment:
This change is not needed, as the tile.visible setter cancels the tileGeometryLoader, the state should be changed at this point.
Maybe its missing in the tile.dispose setter, but basically if the tile becomes "needless" it changes the state of TileGEometryLoader and isExpired can stay the same checking for the state only

Reply:
done